ui/src/app/model/migrate.ts

```diff
@@ -38,4 +38,18 @@ export interface ISourceAndTargetDetails {
     SpannerDatabaseUrl: string
     SourceDatabaseName: string
     SourceDatabaseType: string
+}
+
+export interface ITables {
```

Comment:
I partially agree. Currently `ITables` is semantically equal to a `string[]`. 

We may want to pass more information in the future though so its best to interact between frontend and backend via `Request/Response` objects in my opinion, because otherwise the API signature itself gets modified.